nixos/tests/bees.nix

```diff
@@ -29,27 +29,31 @@ import ./make-test.nix ({ lib, ... }:
 
   testScript =
   let
-    withRetry = content: maxTests: sleepTime: ''
-      max_tests=${lib.escapeShellArg maxTests}; sleep_time=${lib.escapeShellArg sleepTime}; for ((i=0; i<max_tests; i++)); do ${content} && exit 0; sleep "$sleep_time"; done; exit 1;
+    someContentIsShared = loc: pkgs.writeScript "some-content-is-shared" ''
```

Comment:
Can we use `writeShellScript`? It puts the correct shebang right away `runtimeShell`.

Reply:
Done.